### DISCUSSION TITLE
Helm Repository

### DISCUSSION THREAD

**Discussion by h-wb** [jeremylvln/Shulker#335] [Ideas]
Hey! Great work on this! 😃 

I was wondering if you could publish a helm repo for the charts. It makes it easier to get it from a flux managed repo. I made a quick and dirty one myself using an action on [a fork ](https://github.com/h-wb/Shulker/blob/main/.github/workflows/charts-release.yaml)

**Comment by jeremylvln** [comment]
Hi!

Thank you a lot! And thanks for your fork, it seems pretty clean. Give me some time to research the best ways to do it, just for my personal knowledge, and I'll get back to you very shortly :)

Thanks again!

**Reply by jeremylvln** [reply]
I've not yet tested a real installation, so if you are quicker than me to test:

```
$ helm repo add shulker https://shulker.jeremylvln.fr/helm-charts
"shulker" has been added to your repositories

$ helm search repo shulker
NAME                            CHART VERSION   APP VERSION     DESCRIPTION                                       
shulker/shulker-operator        0.1.0           0.5.1           A Kubernetes operator for managing complex and ...
```

**Reply by h-wb** [reply]
Works perfectly, thank you! 🙏
